src/engraving/libmscore/spanner.cpp

```diff
@@ -1236,6 +1236,22 @@ void Spanner::setTicks(const Fraction& f)
     }
 }
 
+void Spanner::setSpecificVoiceIfNeeded(ChordRest* startCR)
```

Comment:
This method is confusing. It takes ChordRest as an argument but has "setSpecific**Voice**" in its name. With this name, I expect to see voice_idx_t as the argument. Also, it should probably be declared in notationinteraction.cpp, not in this base class

Reply:
Ok, I've tidied things up a bit, I think it makes more sense now. Thanks 👍